lib/connectors/email.ts

```diff
@@ -1,4 +1,4 @@
-import { ChainNotConfiguredError, createConnector, normalizeChainId } from "@wagmi/core";
+import { ChainNotConfiguredError, createConnector, normalizeChainId } from "wagmi";
```

Comment:
@LiorAgnin why `@wagmi/core` import is updated?

Reply:
My IDE could not recognize it, only after this change it worked.

Reply:
@LiorAgnin maybe close and reopen your IDE, as `@wagmi/core` is correct:
https://github.com/Web3Auth/web3auth-wagmi-connector/blob/f33ddbb9f66ff5c052b0b06e0b40829bc9d83f17/src/lib/connector.ts#L1

Reply:
I don't think it matters, both ways work well.